README.md

```diff
@@ -11,8 +11,8 @@ wort.schule uses [Ruby on Rails](https://rubyonrails.org/) as a framework and [P
  (e.g. use [asdf](https://asdf-vm.com/))])
 4. Install vips
  (e.g. `brew install vips` on macOS with [Homebrew](https://brew.sh))
-5. Run `bin/setup` (der entsprechende User benötigt Superuser Rechte für diesen Schritt)
-6. Run `bin/rails db:create db:migrate`
+5. Run `bundle install`
+6. Run `bin/rails db:setup`
```

Comment:
Not running `db:setup` was intentional. You can do it, but you'll end up having an empty database, because it only loads the schema, but not the migrations. Within the migrations we load the old data.

This was documented in the change below. Unfortunately the reasoning is missing there.

Reply:
Btw, if `bin/setup` doesn't work, we should fix it. What was the error for you?

Reply:
But `db:setup` also runs the migrations. It's a shortcut for `db:create db:schema:load db:migrate` :) 

Now it works. Not sure, what the error was, can't reproduce it. 

Should I undo the change in the readme?

Reply:
> But `db:setup` also runs the migrations. It's a shortcut for `db:create db:schema:load db:migrate` :)

Hmm, doesn't look that way, cf. https://github.com/rails/rails/blob/caaac482d5f430b29bb4c80e780fa70e65017755/activerecord/lib/active_record/railties/databases.rake#L385

Where did you get the information that `db:migrate` is called as well?

When you do a `db:setup`, do you have data in your `words` table?

> Now it works. Not sure, what the error was, can't reproduce it.
> 
> Should I undo the change in the readme?

If `bin/setup` works, yes, I think we should stay with that as it is the standard for newly created Rails apps.

Reply:
I googled it and it was stated somewhere. Maybe that behavior changed over time. The code doesn't lie and thus you're right :) I reset the changes in the README.md, thanks!